### PR TITLE
feat: Log Meltano version and Operating System at invocation of CLI

### LINKED ISSUE #6939: feature: Print/log Meltano version at the start of execution
### Feature scope

CLI (options, error messages, logging, etc.)

### Description

Either at the DEBUG or INFO level, a line at the very top of the output indicating the version of Meltano, and possibly the operating system, would make it easier for the team and community to help users troubleshoot their issues.

### PR DESCRIPTION
**Pull request by ashu565**
-Added Meltano and Operating System Version at invocation of CLI.(Logger debug level)

### CODE REVIEW

File: src/meltano/cli/__init__.py [hunk 1]
```diff
@@ -97,6 +100,8 @@ def _run_cli():
     Raises:
         KeyboardInterrupt: if caught.
     """
+    logger.debug(f"meltano, version : {__version__}")
+    logger.debug(f"operating system : {os.name}")
```
Comment (edgarrmondragon) [inline src/meltano/cli/__init__.py:104]: This is probably too early. At this point the logging configuration hasn't been loaded. A better place is probably right after https://github.com/meltano/meltano/blob/6c886f89f3dbc61ee1b64d98187c9aaaf7c3d436/src/meltano/cli/cli.py#L113.

Comment (edgarrmondragon) [inline src/meltano/cli/__init__.py:104]: printf-style is preferred for logs, and we can probably emit a single event:

```python
logger.debug("meltano %s, %s", __version__, os.name)
```

https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/

**Review by edgarrmondragon — CHANGES_REQUESTED** [review]

**Comment by edgarrmondragon** [comment]
Thanks @ashu565! I requested some changes.

File: src/meltano/cli/__init__.py (hunk 1 above)
Comment (ashu565) [reply src/meltano/cli/__init__.py]: done

Comment (ashu565) [reply src/meltano/cli/__init__.py]: done

**Comment by edgarrmondragon** [comment]
@ashu565 Can you fix the failing tests?

**Comment by edgarrmondragon** [comment]
@ashu565 Are you still interested in working on this PR?

**Pull request merged**